jOOQ-meta/src/main/java/org/jooq/util/cubrid/CUBRIDTableDefinition.java

```diff
@@ -110,7 +110,7 @@ public CUBRIDTableDefinition(SchemaDefinition schema, String name, String commen
 			    null
 		    );
 
-			result.add(column);
+			result.add(applySyntheticIdentities(column));
 		}
```

Comment:
This is easily forgotten, and I think it is thus applied at the wrong place. A column should always be produced with a synthetic identity already in place. Will try to find a better implementation